packages/webpack/test/aa.spec.js

```diff
@@ -59,23 +59,26 @@ test('aa - generateIdentifierLookup', (t) => {
         },
       },
     },
-    canonicalNameMap: new Map(Object.entries({
-      '/home/user/app': '$root$',
-      '/home/user/app/node_modules/leftpad/': 'leftpad',
-      '/home/user/app/node_modules/fast-json-patch': 'fast-json-patch',
-      '/home/user/app/node_modules/fast-json-patch/module': 'fast-json-patch',
-      '/home/user/app/node_modules/@ethereumjs/util/dist/': '@ethereumjs/util',
-      '/home/user/app/node_modules/@ethereumjs/util/node_modules/ethereum-cryptography/dist/':
-        '@ethereumjs/util>ethereum-cryptography',
-    })),
+    canonicalNameMap: new Map(
+      Object.entries({
+        '/home/user/app': '$root$',
+        '/home/user/app/node_modules/leftpad/': 'leftpad',
+        '/home/user/app/node_modules/fast-json-patch': 'fast-json-patch',
+        '/home/user/app/node_modules/fast-json-patch/module': 'fast-json-patch',
+        '/home/user/app/node_modules/@ethereumjs/util/dist/':
+          '@ethereumjs/util',
+        '/home/user/app/node_modules/@ethereumjs/util/node_modules/ethereum-cryptography/dist/':
+          '@ethereumjs/util>ethereum-cryptography',
+      })
+    ),
```

Comment:
if possible, I'd omit `/home/user/app` and just use `/` as `$root$`

Reply:
okay